swift/build_ffi.sh

```diff
@@ -42,5 +42,13 @@ done
 
 check_rust
 
+if [[ -n "${DEVELOPER_SDK_DIR:-}" ]]; then
+  # Assume we're in Xcode, which means we're probably cross-compiling.
+  # In this case, we need to add an extra library search path for build scripts and proc-macros,
+  # which run on the host instead of the target.
+  # (macOS Big Sur does not have linkable libraries in /usr/lib/.)
+  export LIBRARY_PATH="${DEVELOPER_SDK_DIR}/MacOSX.sdk/usr/lib:${LIBRARY_PATH:-}"
```

Comment:
Why `${LIBRARY_PATH:-}` and not just `${LIBRARY_PATH}` - both have same effect if unset right? Just wondering if I am missing something about this.

Reply:
`set -u` at the top of the script makes bash error on undefined environment variables, but otherwise yeah.